packages/common/src/discussions/HubDiscussions.ts

```diff
@@ -0,0 +1,166 @@
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
```

Comment:
Not sure if we've established this as a definitive file-structure pattern, but for all other entities, we've separated this logic into two separate `edit.ts` and `fetch.ts` modules. Not saying it's necessary to do here, but it might be nice for consistency between entities.

Reply:
Thanks for pointing that out, was following the conventions of the Initiative Entity. Happy to split this into separate `edit` and `fetch` modules, as I found the naming of this file confusing compared to `HubDiscussion.ts`.

Reply:
oh, you're totally right...it looks like what I suggested is only the convention for projects. I agree that the naming is a bit less confusing though

Reply:
I think Tom's intent with edit vs fetch was to have better treeshaking in the future...